meinberlin/apps/newsletters/views.py

```diff
@@ -92,6 +94,10 @@ def form_valid(self, form):
             emails.NewsletterEmail.send(instance,
                                         participant_ids=list(participant_ids),
                                         **self.get_email_kwargs())
+            messages.success(_('Newsletter has been queued to be send.'))
```

Comment:
I don't think the initiator has to know it's a queue, wouldn't be sure how to translate it either and would prefer something as simple as "The newsletter has been saved and will be sent within the next minutes." Or whatever time unit is appropriate.